### PR TITLE
update variable name

### PR DESCRIPTION
**Pull request by Mbaoma**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ def main():
         try:
             while True:
                 ping_opsgenie_heartbeat()
-                time.sleep(PING_SLEEP) 
+                time.sleep(PING_SLEEP_MINUTES)
```
Comment (venkatamutyala) [inline main.py:85]: @Mbaoma this is seconds, isn't it?

Comment (Mbaoma) [reply main.py:85]: on opsgenie, the time format is minutes, hours, days, so i can make this variable PING_SLEEP_SECONDS with a value of 180 on the Values.yaml file in the other repo, what do u think

Comment (venkatamutyala) [reply main.py:85]: Yes my comment was prompted more around you made a commit changing it to: `*_MINUTES` roughly 37 minutes ago  ref: https://github.com/GlueOps/cluster-monitoring/pull/2/commits/31f63e521c950099c2bb198de5b5491d3791442f and 33 minutes ago you requested a review. Assuming i read the python documents correctly the time.sleep() takes seconds as the input hence my suggestion. But yes `PING_SLEEP_SECONDS` sounds good to me.

Comment (Mbaoma) [reply main.py:85]: just saw your other comment, i'd edit

**Pull request closed without merging**